site/gatsby-site/src/components/landing/Featured.js

```diff
@@ -22,45 +23,49 @@ export default function Featured() {
   return (
     <Card className="text-center">
       <Card.Body>
-        <Card.Title as="h2">The Database in Print</Card.Title>
+        <Card.Title as="h2">
+          <Trans ns="landing">The Database in Print</Trans>
```

Comment:
This is not really related to this PR, but I have been wondering whether "in Print" is really the appropriate term to use here. It seems like we're linking to online coverage, so something like "in the Press" would be more accurate.